Dockerfile-rest

```diff
@@ -0,0 +1,26 @@
+FROM ensembl-base
```

Comment:
Why separate the base from the second half?

Reply:
The main reason was build times,  it takes ~ 1hr to build the base.   By separating into two steps, we can deploy different, relatively lightweight rest endpoints